### ISSUE TITLE
Change backlight without writing EEPROM

### ISSUE THREAD

**Issue by sneakin** [adafruit/Adafruit-USB-Serial-RGB-Character-Backpack#4]
I want to change the backlight's color with the outside temperature. Looking at the code I noticed that the EEPROM gets written when I change the command. I'm not going to do the math, but this'll eventually burn it out sooner than later.

**Comment by darconeous** [comment]
Indeed. It seems curious to have done this, considering that the original command set included two versions of the command: one that saved to EEPROM and one that didn't.

**Comment by togaen** [comment]
Yeah I want the backlight to always have a specific color when it is powered on, regardless of what color it was when it was powered off. That doesn't seem possible with this instruction set, which is strange and inconvenient.

**Comment by sneakin** [comment]
Fixed in the https://github.com/transmogrifox/Adafruit_USB_Backpack_Plus fork.
